SPECS/kernel/config

```diff
@@ -2495,6 +2495,16 @@ CONFIG_DM_VERITY_FEC=y
 # CONFIG_DM_INTEGRITY is not set
 # CONFIG_DM_ZONED is not set
 # CONFIG_DM_AUDIT is not set
+CONFIG_DM_IMA_MEASURE_CACHE=y
```

Comment:
As a note, these came in as a result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/3c9e5e21eaf9e892c5d05f6e37b73eaae4292548#diff-c3fca12eac6a48d86acd5db803ae92c8054a6b7edbad5eb336fd66693918498e